docs/ADD_NEW_EXCHANGE.md

```diff
@@ -828,23 +840,23 @@ Run gocryptotrader with the following settings enabled in config
 
 ```go
 // wsReadData gets and passes on websocket messages for processing
-func (f *FTX) wsReadData() {
-	f.Websocket.Wg.Add(1)
-	defer f.Websocket.Wg.Done()
+func (e *Exchange) wsReadData() {
```

Comment:
Note: Should be moving over the multiconnection websocket and this can be removed but I think will defer that to another time.